packages/paste-website/src/pages/primitives/combobox-primitive/index.mdx

```diff
@@ -21,60 +24,38 @@ import {
   defaultExample,
   multiSelectExample,
 } from '../../../component-examples/ComboboxPrimitiveExamples';
-import Changelog from '@twilio-paste/combobox-primitive/CHANGELOG.md';
-
-export const pageQuery = `
-  {
-    allPastePrimitive(filter: {name: {eq: "@twilio-paste/combobox-primitive"}}) {
-      edges {
-        node {
-          name
-          description
-          status
-          version
-        }
-      }
-    }
-    mdx(frontmatter: {slug: {eq: "/primitives/combobox-primitive/"}}) {
-      fileAbsolutePath
-      frontmatter {
-        slug
-        title
-      }
-      headings {
-        depth
-        value
-      }
-    }
-    allAirtable(filter: {data: {Feature: {eq: "Combobox Primitive"}}}) {
-      edges {
-        node {
-          data {
-            Documentation
-            Figma
-            Design_committee_review
-            Engineer_committee_review
-            Code
-            status
-          }
-        }
-      }
-    }
-  }
-`;
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Combobox Primitive');
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
 
 <NormalizedComponentHeader
   categoryRoute={SidebarCategoryRoutes.PRIMITIVES}
   githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/primitives/combobox"
   storybookUrl="/?path=/story/primitives-combobox--dropdown-combobox"
   data={props.data}
+  description={meta.description}
```

Comment:
You can delete all of these. That info is coming from the their `package.json` in getStaticProps.

Reply:
Oh, I noticed that sometimes (like for Box and Package) we wrote a better description that the one in the package.json. Maybe I should just make the package one the same as the one in the mdx?

Reply:
Yeah that would probably be best 👍